data/json/furniture.json

```diff
@@ -1544,7 +1544,7 @@
     "type": "furniture",
     "id": "f_crate_c",
     "name": "crate",
-    "description": "What's inside?  Find out!",
+    "description": "What's inside?  Break it or pry it open to find out!",
```

Comment:
May want to reference that `break`ing crates can damage or destroy items inside. Or maybe that doesn't happen anymore, but it used to (if it doesn't anymore, then this comment is irrelevant!).

Reply:
Or maybe we want the player to learn that the hard way? Since it could be considered obvious?

Reply:
Personally, if players are reading the descriptions, they *should* be rewarded for taking that effort. Meta-wise, it is fairly easy for players to discover out of game, no reason to punish them for it.